src/commands/CommandMusicPlay.java

```diff
@@ -34,19 +44,50 @@ public void action(){
 
 			}
 
-			if(getContent() != null){
-				MusicManager.get().loadTrack(this, getContent());
-			}
-			else{
+			if (getContent() != null) {
+				String id;
+				String source;
+				try {
+					if (!isUrl(getContent())) {
+						
+						YouTube youtube = new YouTube.Builder(new NetHttpTransport(), new JacksonFactory(), new HttpRequestInitializer() {
+							public void initialize(HttpRequest request) throws IOException {
+							}
+						}).setApplicationName("Discord Bot").build();
+
+						YouTube.Search.List search = youtube.search().list("snippet");
+
+						search.setMaxResults((long) 1);
+						search.setQ(getContent());
+						search.setKey(Environment.getVar("YOUTUBE_TOKEN"));
+
+						SearchListResponse response = search.execute();
+
+						id = response.getItems().get(0).getId().getVideoId();
+
+						source = "https://www.youtube.com/watch?v=" + id;
+
+					} else {
+						source = getContent();
+					}
+					
+					MusicManager.get().loadTrack(this, source);
+				
+				} catch (IOException e) {
+				
+					sendMessage("SongByStringFail");
```

Comment:
Seems like you wanted to call the lang line `SongByStringFail` here, but forgot to wrap your string with a `lang("SongByStringFail")`.

---

This makes me realize we should probably have a method like `sendMessageFromLang(String langKey)`, this would help in cases like this one where we only need to send the string through a message without formatting... I'll create an issue for that! :D